src/components/Page2.jsx

```
@@ -27,6 +36,116 @@ const Page2 = () => {
+          setError("Session invalide. Veuillez recommencer depuis la première étape.");
+          setTimeout(() => {
+            navigate("/");
+          }, 3000);
+        } else if (err.response.status === 400 && err.response.data.message?.includes("Email already registered as a Patient")) {
+          setError("Cette adresse e-mail est déjà enregistrée en tant que patient. Veuillez utiliser une autre adresse.");
+        } else if (err.response.status === 400 && err.response.data.message?.includes("Email already registered")) {
+          setError("Cette adresse e-mail est déjà enregistrée. Veuillez utiliser une autre adresse.");
+        } else if (err.response.status === 500) {
+          setError("Erreur serveur. Veuillez contacter l'administrateur ou réessayer plus tard.");
+        } else if (err.response.data && err.response.data.message) {
+          setError(err.response.data.message);
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)